File: pgm/ABC_TypeScript/ABC_185_A.ts
import fs = require("fs");

class Solver
{
    A: number[];
    line: string[];

    input: string;

    constructor()
    {
        this.input = fs.readFileSync("/dev/stdin", "utf8");
        this.line = this.input.split("\n");
        this.A = this.line[0].split(" ").map((str) => parseInt(str, 10));
    }

    solve()
    {
        console.log(this.A.reduce((prev, curr) => Math.min(prev, curr)));
        return 0;
    }
}

const s = new Solver();
s.solve();

File: pgm/ABC_Dart/ABC_185_A.dart
// import 'dart:collection';
// import 'dart:convert';
import 'dart:io';
import 'dart:math';
// import 'dart:typed_data';

void main() {
  print(stdin.readLineSync()!.split(' ').map(int.parse).reduce(min));
  exit(0);
}
